WorldMoodTracker/js/components/WorldMoodTracker.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import Header from './Header';
+import WorldMap from './WorldMap';
+import Footer from './Footer';
 
 export default class WorldMoodTracker extends React.Component {
     render() {
         return (
             <div>
-                <h1>
-                    World Mood Tracker
-                </h1>
+                <Header/>
+                <WorldMap/>
+                <Footer />
             </div>
```

Comment:
I didn't look into it that way. I kept it at the bottom in `xs` because there would be no space in the header. Also, I didn't wanted a hamburger menu that would draw on the map.